apps/antalmanac/src/lib/termData.ts

```diff
@@ -112,7 +112,7 @@ function getFinalsStartForTerm(term: string) {
  */
 function getDefaultFinalsStartDate() {
     // FIXME: Un-offset once Spring starts, or figure out a proper fix
-    const [year, month, day] = termData[defaultTerm + 1].finalsStartDate || [];
+    const [year, month, day] = getDefaultFinalsStart() || [];
```

Comment:
```suggestion
    const [year, month, day] = getDefaultFinalsStart() ?? [];
```
Prefer nullish coalescing to an `or`, since nullish coalescing only falls back to the right value if the left is explicitly null or undefined, while `or` will fallback whenever it's falsy. Falsy values include empty strings, `0`, etc. which might be valid values and not the same as null/undefined.